### PR TITLE
new-app: Add image tag in generated imageStream

### PR DESCRIPTION
**Pull request by kargakis**
@smarterclayton PTAL

### CODE REVIEW

File: pkg/generate/app/app.go [hunk 1]
```diff
@@ -259,6 +259,9 @@ func (r *ImageRef) ImageStream() (*imageapi.ImageStream, error) {
 		ObjectMeta: kapi.ObjectMeta{
 			Name: name,
 		},
+		Spec: imageapi.ImageStreamSpec{
+			Tags: map[string]imageapi.TagReference{r.Tag: {DockerImageReference: r.DockerImageReference.String()}},
```
Comment (smarterclayton) [inline pkg/generate/app/app.go]: The string value should be "latest" - we don't want require admins to guess what the tag they created was.  There's very little value in generating to the remote one.

Comment (ncdc) [inline pkg/generate/app/app.go]: imageapi.DefaultImageTag instead?

**Comment by kargakis** [comment]
[test]

**Comment by smarterclayton** [comment]
LGTM [merge] thanks

**Pull request merged**